.changeset/angry-spoons-grin.md

```diff
@@ -0,0 +1,9 @@
+---
+"@primer/react": patch
+---
+
+- Add a block prop for full width
+- Add alignContent prop to align content to center or start
+- Use control sizing CSS variable values (not using CSS vars just yet)
+- Use height over padding for more control over sizing
```

Comment:
It's a bit easier to manage control size with static `height` and `width` _along_ with internal padding. Its especially apparent for IconOnly buttons. For buttons, since we never want a line break using height feels like a safe solution.

Reply:
Oh I didn't realize we never allowed a line break in buttons.

When the button text doesn't fit on one line, do we: 
- automatically truncate to a single line?
- require the user to handle truncation?
- something else?

Reply:
We haven't implemented anything to manage overflow, its just a pattern recommendation to keep labels succinct. I'm not opposed to adding an ellipses or something. 

![do and dont image showing a correct button with a single line label, and a multi line label in the "dont" example](https://user-images.githubusercontent.com/18661030/208536614-af7e2877-b829-4323-84e5-a98aaa4f2796.png)